readme.md

```diff
@@ -1,49 +1,63 @@
-
-**Install CMake** from the [CMake download page](http://www.cmake.org/download/), or using your system's package manager. Make sure to check the option in the installer to add it to your path. 
+**Tip:** If permission is denied on Linux or OS X, you may need to run ``sudo pip install yotta``. 
 
-**Which compiler** you need depends on whether you're building programs for your host system, or cross-compiling them run on an embedded device:
+### Installation - CMake
 
- * to cross-compile, install [arm-none-eabi-gcc](https://launchpad.net/gcc-arm-embedded/+download).
- * to compile natively on OS X, [install Xcode](https://developer.apple.com/xcode/downloads/), including the command-line tools.
- * to compile natively on Linux, install clang with your system's package manager.
+**Install CMake** from the [CMake download page](http://www.cmake.org/download/), or using your system's package manager. Make sure to check the option in the installer to add it to your path. 
 
-Further information on installing yotta for different platforms can be found on the [documentation site](http://docs.yottabuild.org/#installing).
+### Installation - Compilers
 
-### Get Started!
-The best way to get started is to [follow the tutorial](http://docs.yottabuild.org/tutorial/tutorial.html).
+**Which compiler** you need depends on whether you're building programs for your host system, or cross-compiling them to run on an embedded device:
 
-### What `yotta` does
-yotta downloads the software components that your program depends on (it's similar in concept to npm, pip or gem). To install a new module, you run `yotta install --save <modulename>`, and yotta will install both the module you've specified and any of its dependencies that you don't already have installed, and save the fact that you depend on that module into your module's description file.
+ * To cross-compile, install [the **arm-none-eabi-gcc**](https://launchpad.net/gcc-arm-embedded/+download).
+ * To compile natively on OS X, [install **Xcode**](https://developer.apple.com/xcode/downloads/), including the command-line tools.
+ * To compile natively on Linux, install **Clang** with your system's package manager.
 
-To really understand how yotta works, you should install yotta (see above), then [follow the tutorial](http://docs.yottabuild.org/tutorial/tutorial.html).
+Further information on installing yotta for different platforms can be found on the [documentation site](http://docs.yottabuild.org/#installing).
 
 ### Further Documentation
+
 For further documentation see the [yotta docs](http://armmbed.github.io/yotta/) website.
 
 ### Tips
- * `yt` is a shorthand for the `yotta` command, and it's much quicker to type!
- * yotta is strongly influenced by [npm](http://npmjs.org), the awesome node.js software packaging system. Much of the syntax for module description and commands is very similar.
+
```

Comment:
please avoid unnecessary white-space changes, makes it hard to see what's changed :stuck_out_tongue_winking_eye: